src/Ng/Jobs/Db2CatalogJob.cs

```diff
@@ -119,8 +122,23 @@ protected override void Init(IDictionary<string, string> arguments, Cancellation
             Destination = catalogStorageFactory.BaseAddress;
             TelemetryService.GlobalDimensions[TelemetryConstants.Destination] = Destination.AbsoluteUri;
 
-            Top = 20;
-            Timeout = TimeSpan.FromSeconds(300);
+            // Setup gallery database access
+            PackageContentUriBuilder = new PackageContentUriBuilder(
+                arguments.GetOrThrow<string>(Arguments.PackageContentUrlFormat));
+
+            var connectionString = arguments.GetOrThrow<string>(Arguments.ConnectionString);
+            GalleryDbConnection = new AzureSqlConnectionFactory(
+                connectionString,
+                new EmptySecretInjector(),
+                LoggerFactory.CreateLogger<AzureSqlConnectionFactory>());
+
+            const int timeoutInSeconds = 300;
```

Comment:
IMO this should be configurable. We've had to plumb this to config/params so many times for other DB jobs (like stats)

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/0a025e89b7819fd5991d4d5980e222acf8deeecb